package.json

```diff
@@ -50,6 +50,7 @@
     "url": "https://github.com/kefranabg/readme-md-generator/issues"
   },
   "homepage": "https://github.com/kefranabg/readme-md-generator#readme",
+  "demoUrl": "https://github.com/kefranabg/readme-md-generator#-demo",
```

Comment:
`demoUrl` is not a field that is usually used in `package.json`. The question about demoUrl should never have default value, user is free to answer the question if he wants to.

Reply:
Thank  You for the code review I´ve removed the demo url default value, let me know if more changes are required :)